script/queries/nfp_audit_report_non_congress.rb

```diff
@@ -26,9 +26,11 @@
 
 emp_ids = Employer.where(:fein => {"$nin" => congress_feins }).map(&:id)
 
+start_year = Date.today.year - 3
 pols = Policy.where({
     :enrollees => {"$elemMatch" => {
-          :rel_code => "self"
+          :rel_code => "self",
+          :coverage_start => {"gt" => Date.new(start_year, 12, 31)},
```

Comment:
The syntax looks good and aligns with our query requirements. Just a small suggestion: ensure all keys are strings for consistency. Thanks for your efforts!
gt should be "$gt" for proper MongoDB syntax.
`"coverage_start" => { "$gt" => Date.new(start_year, 12, 31) }`